fbs/consumer.fbs

```diff
@@ -0,0 +1,15 @@
+
+namespace FBS.Consumer;
+
+table ConsumerLayers {
+  spatial_layer:uint8;
+  temporal_layer:uint8;
+}
+
+table ConsumerScore {
+  score:uint8;
+  producer_score:uint8;
+  producer_scores:[uint8];
+}
+
+root_type ConsumerLayers;
```

Comment:
What is this?

Reply:
Needed by flatbuffers just for the case then the schema represents a restricted JSON, which is not our case. I'll check if it can be removed.